src/vanna/logger.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import sys
+
+
+def get_logger():
+    """
+    Method to get logger.
+
+    Returns:
+        logging: Logger
+    """
+    return logging.getLogger(__name__)
+
+
+def initialize_logger():
+    """
+    Method that defines the standardization of logging.
+
+    Returns:
+        logging: Updated logger
+    """
+    logging.basicConfig(
+        level=logging.INFO,  # default: 1
+        format="[%(asctime)s] [%(filename)s:%(lineno)d] %(levelname)s - %(message)s",
+        handlers=[logging.StreamHandler(stream=sys.stdout)],
+    )
+
+    return get_logger()
+
+
+def setVerbosity(level: str = "INFO"):
```

Comment:
Drop this, we don't want multiple ways of doing things. Users should call `logging.getLogger("vanna").setLevel(level)` instead. Should document this.

Reply:
Sorry, I could have suggested this in a nicer way! I think this is more of a matter of taste, I'm curious if anyone else has thoughts. At the least, I would like this to be `def set_log_level(level: str)` so that there isn't the verbosity/level mismatch

Reply:
My opinions;
 - use snake case, like the rest of codebase
 - `setVerbosity` currently does only 1 thing (1 line), so this as an interface isn't providing a helpful abstraction for anything. If in the future, this function should take care of other things, like pointing to the correct logger, maybe then it might be justified... but that would be building things for a potential use case, not a right-now use case.
 - perhaps later, if a real need arises to have more complicated logging (across multiple libraries etc), then maybe a convenience func could be a nice addition. but best wait for that situation to arise.

I've not been following this PR for long, so please take that into consideration with the above, these opinions may be uninformed!

Reply:
Not much point doing anything here until I hear from @zainhoda, IMO.

We can always sketch some solution that works, but it needs to suit what the Vanna team wants. But I agree with @NickCrews proposals, I am just waiting for instructions.